shimmerlayout/src/main/java/io/supercharge/shimmerlayout/ShimmerLayout.java

```diff
@@ -86,6 +93,16 @@ protected void dispatchDraw(Canvas canvas) {
         }
     }
 
+    @Override
+    public void setVisibility(int visibility) {
+        super.setVisibility(visibility);
+        if (visibility == VISIBLE) {
```

Comment:
It is true, than just format to the second if() that the startShimmerAnimation() would be in a new line.